.changes/v2.15.0/434-features.md

```diff
@@ -0,0 +1 @@
+* Add environment variable `GOVCD_API_VERSION` so API version can be set manually [GH-434]
```

Comment:
I think we need a section in TESTING.md (or somewhere similar) on how to use it and how not to use this new variable. Someone may think that it's as trivial as changing this value to increase the API version, but in reality that has many consequences. I think it would be good to write it up.

CC: @Didainius @adambarreiro

Reply:
Yes, I agree, we have to document this

Reply:
Done, please check 😄 
https://github.com/vmware/go-vcloud-director/pull/434/files#diff-4da5f2fd7bc2f11672282410e99aad91d422f9f4fb6c723abc7ef29ee5cb6906R345-R346